### PR TITLE
Refs #476 -- Added tutorials for using WalletConnect in Next.js project

### PR DESCRIPTION
**Pull request by jackleeio**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: #476  (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/532)

### CODE REVIEW

**Review by darwintree — CHANGES_REQUESTED** [review]
Several changes are requested. Besides, a relating repo is created (https://github.com/conflux-fans/web3modal-conflux-nextjs) with vercel deployments.

**Comment by jackleeio** [comment]
> Several changes are requested. Besides, a relating repo is created (https://github.com/conflux-fans/web3modal-conflux-nextjs) with vercel deployments.

A related project already exists, we created it a few days ago（https://github.com/ConfluxDAO/web3modal-conflux-nextjs)

File: docs/espace/tutorials/walletConnect/nextjs.md [hunk 1]
```diff
@@ -1,11 +1,220 @@
...
+
+## Step 1: Set Up Your Next.js Project
+
+First, create a new Next.js application if you haven't already:
+
+```bash
+npx create-next-app web3modal-conflux-nextjs
+cd web3modal-conflux-nextjs
+```
+
```
[the remark this answers is not in the document]
Comment (jackleeio) [reply docs/espace/tutorials/walletConnect/nextjs.md:30]: already added

File: docs/espace/tutorials/walletConnect/nextjs.md [hunk 2]
```diff
@@ -1,11 +1,220 @@
...
+```typescript
+import { cookieStorage, createStorage } from "wagmi";
+import { confluxESpace } from "wagmi/chains";
+
+import { http, createConfig, WagmiProvider } from "wagmi";
+import { walletConnect, injected, coinbaseWallet } from "wagmi/connectors";
+
+// Get projectId at https://cloud.walletconnect.com
+export const projectId = process.env.NEXT_PUBLIC_PROJECT_ID;
```
[the remark this answers is not in the document]
Comment (jackleeio) [reply docs/espace/tutorials/walletConnect/nextjs.md:53]: already added

**Comment by darwintree** [comment]
> > Several changes are requested. Besides, a relating repo is created (https://github.com/conflux-fans/web3modal-conflux-nextjs) with vercel deployments.
> 
> A related project already exists, we created it a few days ago（https://github.com/ConfluxDAO/web3modal-conflux-nextjs)

That's great. Then I would suggest the repo link be added to the article as well as the vercel builds for online preview

**Pull request merged**